kyuubi-server/src/test/scala/org/apache/kyuubi/engine/trino/TrinoProcessBuilderSuite.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.engine.trino
+
+import org.apache.kyuubi.KyuubiFunSuite
+import org.apache.kyuubi.KyuubiSQLException
+import org.apache.kyuubi.config.KyuubiConf
+import org.apache.kyuubi.config.KyuubiConf.ENGINE_TRINO_CONNECTION_CATALOG
+import org.apache.kyuubi.config.KyuubiConf.ENGINE_TRINO_CONNECTION_URL
+
+class TrinoProcessBuilderSuite extends KyuubiFunSuite {
+  private def conf = KyuubiConf().set("kyuubi.on", "off")
+
+  test("trino process builder") {
+    val trinoConf = conf
+      .set(ENGINE_TRINO_CONNECTION_URL, "http://0.0.0.0:8080")
```

Comment:
We need get the address and port from the started container

Reply:
e.g. https://github.com/housepower/spark-clickhouse-connector/blob/47430ff7c2c6047575287e6ee4fb7f273bcc33c9/clickhouse-core-integration/src/testFixtures/scala/xenon/clickhouse/base/ClickHouseSingleMixIn.scala#L52

Reply:
It is used to test process builder, not real connect, not necessary here. Removed